### PR TITLE
SQL Server REAMDE docs

### PR DESCRIPTION
**Pull request by pranshi06**
Update Readme docs

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,157 +1,138 @@
...
-### Compile
+[![Docs](https://img.shields.io/badge/docs-v3.x-brightgreen.svg?style=flat)](https://hasura.io/docs/3.0/getting-started/overview/)
+[![ndc-hub](https://img.shields.io/badge/ndc--hub-sqlserver-blue.svg?style=flat)](https://hasura.io/connectors/sqlserver)
+[![License](https://img.shields.io/badge/license-Apache--2.0-purple.svg?style=flat)](LICENSE.txt)
+[![Status](https://img.shields.io/badge/status-alpha-yellow.svg?style=flat)](./readme.md)
 
-```
-cargo build
-```
+The Hasura SQLServer Connector allows for connecting to a SQL Server database giving you an instant GraphQL API on top
```
Comment (rsd1122) [inline readme.md:8]: With this connector, Hasura allows you to instantly create a real-time GraphQL API on top of your data models in Microsoft SQL Server. This connector supports SQL Server's functionalities listed in the table below, allowing for efficient and scalable data operations. Additionally, users benefit from all the powerful features of Hasura’s Data Delivery Network (DDN) platform, including query pushdown capabilities that delegate query operations to the database, thereby enhancing query optimization and performance.

File: readme.md [hunk 2]
```diff
@@ -1,157 +1,138 @@
...
+of your SQL Server data.
```
Comment (rsd1122) [inline readme.md:9]:
```suggestion
```

File: readme.md [hunk 3]
```diff
@@ -1,157 +1,138 @@
...
+1. Create a [Hasura Cloud account](https://console.hasura.io)
+2. Install the [CLI](https://hasura.io/docs/3.0/cli/installation/)
+3. Install the [Hasura VS Code extension](https://marketplace.visualstudio.com/items?itemName=HasuraHQ.hasura)
+4. [Create a project](https://hasura.io/docs/3.0/getting-started/create-a-project)
 
-See [debugging.md](./debugging.md).
+## Using the connector
 
-### Profile
+To use the SQL server connector, follow these steps in a Hasura project:
```
Comment (rsd1122) [inline readme.md:49]:
```suggestion
To use the SQL Server connector, follow these steps in a Hasura project:
```

File: readme.md [hunk 4]
```diff
@@ -1,157 +1,138 @@
...
+5. Test the API
 
-## Linting
+   The console URL in the build information cna be used to open the GraphiQL console to test out the API
 
-Run `just lint` to run clippy linter
+## Contributing
 
-run `just lint-apply` to attempt to autofix all linter suggestions
+ndc-sqlserver is still in early stages of development and we are currently not accepting contributions.
```
Comment (rsd1122) [inline readme.md:112]: Why?? We should accept all contributions right @gneeri ?

File: readme.md [hunk 5]
```diff
@@ -1,157 +1,138 @@
...
 
-## Formatting
+## License
 
-Check your formatting is great with `just format-check`.
+The Hasura SQLServer connector is available under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0).
```
Comment (rsd1122) [inline readme.md:116]:
```suggestion
The Hasura SQL Server connector is available under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0).
```

File: readme.md [hunk 6]
```diff
@@ -1,157 +1,138 @@
...
 
-Format all Rust code with `just format`.
+## Running the connector locally
+
+To run the SQL server connector locally, follow these steps:
+
+1. Initialize a configuration:
```
Comment (rsd1122) [inline readme.md:122]:
```suggestion
1. Set up a connector configuration file:
```

**Review by rsd1122 — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (pranshi06) [reply readme.md]: Updated

File: readme.md (hunk 4 above)
Comment (gneeri) [reply readme.md]: Yeah, I think if someone were willing to put in the time, we'd be happy for the contribution. Perhaps we could just say something like:

`ndc-sqlserver is still in early stages and we have some work actively underway, but we are always happy for any community contributions.`

**Review by gneeri — APPROVED** [review]

**Pull request merged**